app/models/event.rb

```diff
@@ -15,6 +15,32 @@ class Event < ActiveRecord::Base
 
   validates :max_participants, numericality: { only_integer: true, greater_than: 0 }
 
+  # Returns whether all application_letters are classified or not
+  #
+  # @param none
+  # @return [Boolean] if status of all application_letters is not equal 2
+  def applicationsClassified?
+    application_letters.all? { |application_letter| application_letter.status != 'pending' }
+  end
+
+  # Returns a string of all email addresses of rejected applications
+  #
+  # @param none
+  # @return [String] Concatenation of all email addresses of rejected applications, seperated by ','
+  def compute_rejected_applications_emails
```

Comment:
I think the name doesn't fit very well. "Compute" sounds for me like a time consuming task with a fancy algorithm. Moreover this method doesn't compute/generate the rejected emails. But it is indeed not so easy to find a proper name...